app.json

```diff
@@ -56,6 +56,8 @@
     "RAILS_SERVE_STATIC_FILES": {
       "required": true
     },
+    "RECAPTCHA_SECRET_KEY": "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe",
```

Comment:
Thanks for checking! But yes, those are special "test keys" google provides for this purpose.